File: modul 7/soal 1/PRAK601-2210817220034-Wanda Karliyanti.c
#include<stdio.h>

int main(){
int b, k, i, j, a;
scanf("%d %d", &b, &k);

int angka[b][k];
for(i=0;i<b;i++){
    for(j=0;j<k;j++){
        scanf("%d", &a);
        angka[i][j]=a;
    }
}
printf("\n");
for(i=0;i<b;i++){
    for(j=0;j<k;j++){
        printf("%d ", angka[i][j]);
    }
    printf("\n");
}
}

File: modul 7/soal 1/PRAK601-2210817220034-Wanda Karliyanti.py
b, k=map(int, input().split())
angka=[]
x=0
a=list(map(int, input().split()))
for i in range (b):
    angka.append(a[x:x+k])
    x=x+k
print("")
for b in angka:
    for a in b:
        print(a, end=' ')
    print("")